web/src/compose_actions.ts

```diff
@@ -455,6 +459,7 @@ export let cancel = (): void => {
     compose_state.set_message_type(undefined);
     compose_pm_pill.clear();
     $(document).trigger("compose_canceled.zulip");
+    compose_tooltips.hide_go_to_conversation_button_intro_tooltip();
```

Comment:
Here, we are calling it every time as well.

Reply:
This is now fixed. Thank you.